source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -4,40 +4,40 @@ Skip Returned Results
 
 .. default-domain:: mongodb
 
-In this guide, we show you how to skip returned results.
+In this guide, we show you how to skip a specified number of returned results.
 
-You can skip results using the ``skip()`` method of a 
-:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`
-instance or by using the :java-docs:`Aggregates.skip() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>` 
-static factory method in an aggregation pipeline. 
+You can skip results using the ``skip()`` method on a result of a query or
+in an aggregation pipeline. 
 
-The ``skip()`` method takes an integer that specifies how many documents
-to omit from the beginning of the list of documents returned by your
-query. 
+The ``skip()`` method takes an integer that specifies the number of documents
+to omit from the beginning of the list of documents returned by the
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`. 
 
-You can use the ``skip()`` method of a ``FindIterable`` instance as follows:
+You can use the ``skip()`` method to skip the first two documents as follows:
 
 .. code-block:: java
 
    collection.find().skip(2); 
 
-The ``Aggregates.skip()`` method implements the ``Bson`` interface to
-specify skip criteria as an argument.  
+The :java-docs:`Aggregates.skip() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#skip(int)>`
+is a stage in the aggregation pipeline where you can specify a skip criteria as an argument.  
```

Comment:
Aggregates.skip() is a builder method, not a stage. You can use Aggregates.skip() to build the Bson that represents an aggregation stage (you can also use other objects like Document or JsonObject), so we shouldn't refer to it as "a stage in the aggregation pipeline". Maybe something like "you can include Aggregates.skip() as a stage in the aggregation pipeline to specify how many documents to omit from the beginning of the results of the prior stage."